hack/swagger.sh

```diff
@@ -0,0 +1,69 @@
+#!/bin/bash
+
+set -o nounset
+set -o errexit
+set -o pipefail
+
+# Get the absolute path of this file
+DIR="$( cd "$( dirname "$0"  )" && pwd  )"
+cd "${DIR}" || return
+
+# Here is details of the swagger binary we use.
+# $ swagger version
+# version: 0.19.0
+SWAGGER_VERSION=v0.19.0
+
+. ./env.sh
+
+BUILD_PATH=../bin/${GOOS}_${GOARCH}
+
+SWAGGER_BIN="swagger"
+
+# swagger::check_version checks the command and the version.
+swagger::check_version() {
```

Comment:
I am wondering if this part of code is necessary. Just use the exit code and stdout to check the existence. Maybe be no need to use `has_installed`. From the meaning of `has_installed`, it should be some type like boolean, while it is a string.